suite-common/suite-utils/src/antiFraud.ts

```diff
@@ -1,8 +1,25 @@
 import BigNumber from 'bignumber.js';
 
-import type { WalletAccountTransaction } from '@suite-common/wallet-types';
+import type { TokenDefinitions, WalletAccountTransaction } from '@suite-common/wallet-types';
 
 export const getIsZeroValuePhishing = (transaction: WalletAccountTransaction) =>
     new BigNumber(transaction.amount).isEqualTo(0) &&
     !!transaction.tokens.length &&
     transaction.tokens.every(token => new BigNumber(token.amount).isEqualTo(0));
+
+export const getIsFakeTokenPhishing = (
+    transaction: WalletAccountTransaction,
+    tokenDefinitions: TokenDefinitions,
+) =>
+    new BigNumber(transaction.amount).isEqualTo(0) && // native currency is zero
+    !!transaction.tokens.length && // there are tokens in tx
+    transaction.tokens.every(tokenTx => ['ERC20', 'SPL'].includes(tokenTx?.standard || '')) && // non-nfts
+    !transaction.tokens.some(tokenTx => tokenDefinitions[tokenTx.contract]?.isTokenKnown); // all are tokens are unknown
+
+export const getIsPhishingTransaction = (
+    transaction: WalletAccountTransaction,
+    tokenDefinitions: TokenDefinitions,
+) =>
+    getIsZeroValuePhishing(transaction) ||
+    (!!Object.keys(tokenDefinitions).length && // at least one token definition is available
```

Comment:
nit I preffer to use guard functions for these common checks. It is shorter and more readable. But I understand if you don't want to apply this. Feel free to resolve.
```suggestion
    import {D} from "@mobily/ts-belt";

    (D.isNotEmpty() && // at least one token definition is available
```

Reply:
84a195fa785048842a2b5cdcf3187cfdf2623cc2

Reply:
Could be applied also few lines above, but I won't block this PR because of that.

Reply:
e7e24be0919bc33b6546171f9d8126e7dea5dd78